### PR TITLE
React issue2

### PR DESCRIPTION
**Pull request by tmcswain**

### CODE REVIEW

**Comment by tmcswain** [comment]
[React Issue 2: Add FeedbackStore, and add FeedbackForm](https://trello.com/c/uQxbr9qn/282-react-issue-2-add-feedbackstore-and-add-feedbackform)

File: app/javascript/image_sharing/test/stores/FeedbackStore.spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import { describe, it, beforeEach } from 'mocha';
+
+import FeedbackStore from '../../stores/FeedbackStore';
+
+describe('FeedbackStore', () => {
```
Comment (natalieschauser) [inline app/javascript/image_sharing/test/stores/FeedbackStore.spec.js]: Can we test the default values of `userName` and `comments`?

File: app/javascript/image_sharing/test/components/FeedbackForm.spec.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    expect(wrapper.find('#js-username-input').props().value).to.equal('Taylor');
+  });
+
+  it('asks for comments', () => {
+    const wrapper = mount(<FeedbackForm stores={{ feedbackStore: store }} />);
+    expect(wrapper.find('.js-comments').text()).to.equal('Comments:');
+    expect(wrapper.find('#js-comments-input').props().value).to.equal('Some Comments');
+  });
+
+  it('updates name onChange', () => {
```
Comment (natalieschauser) [inline app/javascript/image_sharing/test/components/FeedbackForm.spec.js]: Awesome work testing `onChange` of name! Can we do the same for `onChange` of the comment?

File: app/javascript/image_sharing/components/App.js [hunk 3]
```diff
@@ -2,13 +2,15 @@ import React, { Component } from 'react';
 import { inject } from 'mobx-react';
 import Header from './Header';
 import Footer from './Footer';
+import FeedbackForm from './FeedbackForm';
 
 class App extends Component {
```
Comment (natalieschauser) [inline app/javascript/image_sharing/components/App.js]: Can we add a test for `App` to assert that `FeedbackForm` is shown.

**Pull request merged**